sale_pricelist_margin/models/product_pricelist_item.py

```diff
@@ -0,0 +1,24 @@
+from odoo import api, fields, models
+
+
+class ProductPricelistItem(models.Model):
+    _inherit = "product.pricelist.item"
+
+    discount_type = fields.Selection(
+        [("discount", "Discount"), ("sale_margin", "Sale Margin")],
+        default="discount",
+    )
+    price_discount = fields.Float(
+        readonly=False,
+        compute="_compute_price_discount",
+    )
+    sale_margin = fields.Float(
+        help="The margin percentage for sale price.",
+    )
```

Comment:
Hi. looks lighter ! 
I wonder if it's possible to : 
- remove discount_type : 
- keep visible price_discount AND sale_margin and make them computable depending on the other.

only a remarks. not a blocking point.

thanks !